packages/yoga/src/helpers.ts

```diff
@@ -47,3 +50,142 @@ export function invalidateImportCache(files: string[]) {
     }
   })
 }
+
+/**
+ * Find a config file path
+ */
+export function findConfigFile(fileName: string) {
+  return ts.findConfigFile(
+    /*searchPath*/ process.cwd(),
+    ts.sys.fileExists,
+    fileName,
+  )
+}
+
+/**
+ * - Compute paths relative to the root of the project
+ * - Set defaults if some options are missing
+ */
+function normalizeConfig(rootPath: string, config: InputConfig): Config {
+  if (config.resolversPath) {
+    config.resolversPath = relativeToRootPath(rootPath, config.resolversPath)
+  } else {
+    config.resolversPath = relativeToRootPath(rootPath, './src/graphql')
+  }
+
+  // Context path is optional and should remain undefined if none is provided or if the default path doesn't exist
+  if (config.contextPath) {
```

Comment:
[Idea] Maybe it will be better to setup default config object somewhere in code and perform deep-merge of the user provided config and default config (so default entires will be overriden). This will simplify code like here where we need to check for existence and setup defaults. This defaults often repeat themselves in different places. 
With that all 5 lines bellow could become single line + code will be easier to test and maintain. This is just idea - not strong opinion here.